dascore/io/sr4731/utils.py

```
@@ -0,0 +1,315 @@
+    if not data.size:
+        return []
+    attrs["coords"] = cm
+    patch = dc.Patch(data=data, coords=cm, attrs=attr_class(**attrs))
+    return [patch]
+
+
+def get_format(resource, name: str, version: str) -> tuple[str, str] | bool:
+    """Return SR-4731 format/version if the resource is supported."""
+    try:
+        parsed = _parse_sor(resource, load_samples=False)
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)